root/user/UserCollections.js

```diff
@@ -88,7 +88,9 @@ const CollectionsEntityTypeSection = ({
   user,
 }) => {
   const viewingOwnProfile = !!$c.user && $c.user.id === user.id;
-  const nameColumn = defineNameColumn(l('Collection'));
+  const nameColumn =
+    defineNameColumn<CollectionT>(false, '', false, l('Collection'));
```

Comment:
`defineNameColumn` should have optional arguments with default values to avoid such kludge I did.